webapp/lib/model.g.dart

```diff
@@ -653,6 +654,11 @@ typedef UserConfigurationCollectionListener = void Function(
   List<UserConfiguration> removed,
 );
 
+typedef OnErrorListener = void Function(
+  Object error,
+  StackTrace stackTrace
+);
```

Comment:
I like these model changes! Nice work!
Once this lands, I can update model codegen to include these changes.

Reply:
cool